packages/EasyTest/src/Traits/ExceptionTrait.php

```diff
@@ -36,10 +36,12 @@ protected function assertThrownException(
             throw $this->thrownException;
         }
 
-        if (\str_starts_with($expectedException, 'App\\')) {
-            $message = "Didn't you forget to define the error code?";
-            self::assertNotNull($code, $message);
-            self::assertNotSame(0, $code, $message);
+        if (
+            $code === null
```

Comment:
How is this supposed to work? Because here we are checking if `$code` is null but then on line 44 we are asserting that `$code` is not null.

Reply:
Yes, it's not clear but we expect "always true" assert here. Just because we check all that we need in the "if" block. We need to check user provides an error code if `thrownException->getCode() != 0` otherwise shows a message.